### ISSUE TITLE
[Feature Request] Create a request from curl

### ISSUE THREAD

**Issue by smdedar** [usebruno/bruno#654]
User can create a request from curl

```
curl --location 'https://test.com/api/v1/auth/login' \
--header 'Content-Type: application/json' \
--data-raw '{
    "email": "test@exmaple.com",
    "password": "test"
}'
```

**Comment by niqdev** [comment]
I really like the tool and think this feature would be my main reason to move to `bruno`, actually it's really handy because you can get it directly from chrome or firefox 

![Screenshot from 2023-10-27 18-40-23](https://github.com/usebruno/bruno/assets/9458643/bf206be8-b235-40af-9b87-25e56f50c6d2)

**Comment by inoyakaigor** [comment]
Today I tried move from Insomnia to Bruno and failed because of lack of this feature

**Comment by helloanoop** [comment]
@inoyakaigor Curl import feature will be available in 1.1.0 release which will be live in the next few hours.

**Comment by helloanoop** [comment]
@inoyakaigor Curl Import functionality is now available in `1.1.0` 🎉

**Comment by niqdev** [comment]
That works like a charm, thanks!

**Comment by helloanoop** [comment]
Keeping this ticket open.

We need to improve the ux a bit.

If a user directly pastes a curl command into the URL box at these two places.
1. New Request Modal
2. Already opened tab.

Then, the curl import should still go through.

**Comment by asdofindia** [comment]
I just tried bruno for the first time today. And it was a pleasure to add a request from curl exactly as described here. I ran into an issue where the curl command was working, but the bruno request got internal server error from the server. I didn't know this feature was literally implemented a couple of days ago.

But the issue is that the `--data-raw` json didn't get added to the body.

I think parse-curl doesn't have support for that parameter as I don't see any mention of "raw" in https://www.npmjs.com/package/parse-curl?activeTab=code

**Comment by asdofindia** [comment]
There seems to be unreleased code in https://github.com/tj/parse-curl.js which handles `--data-raw` and is last updated 3 years ago (better than 6)

**Comment by inoyakaigor** [comment]
@asdofindia it would be great if teaxarea for curl request will be bigger or can be resizable
![image](https://github.com/usebruno/bruno/assets/1235286/f4c655bf-f489-4853-b225-78c077f0eb03)
because edit query in small textarea is not handy 
![image](https://github.com/usebruno/bruno/assets/1235286/065889b8-fe36-47c1-be12-2135fb83586d)

**Comment by helloanoop** [comment]
@asdofindia @inoyakaigor The issues have been fixed in `v1.1.0`

I am closing this issue. For any issues or feedback on curl, please use https://github.com/usebruno/bruno/issues/338
